test/mocks/server/drain_manager.cc

```diff
@@ -0,0 +1,25 @@
+#include "drain_manager.h"
+
+#include <string>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using testing::_;
```

Comment:
As I mentioned on the last PR, these should be fully-qualified and inside the innermost working namespace, or removed if not being used.

Reply:
Thanks for let me know, now they should be fully qualified and stay inside the innermost namespace.